### PR TITLE
Support for changing fonts

### LINKED ISSUE #74: Customers fonts

### PR DESCRIPTION
**Pull request by villejuutila**
Resolves #74

### CODE REVIEW

File: src/components/layout/v2/button-component-properties.tsx [hunk 1]
```diff
@@ -105,10 +131,24 @@ const ButtonComponentProperties = ({
     updateComponent(component);
   };
 
+  /**
+   * Renders font menu items
+   *
+   * @param font font
+   */
+  const renderFontMenuItem = (font: any) => {
```
Comment (Jdallos) [inline src/components/layout/v2/button-component-properties.tsx:139]: Could this be typed as string?

File: src/components/layout/v2/text-component-properties.tsx [hunk 2]
```diff
@@ -226,6 +256,19 @@ const TextComponentProperties = ({
     </ToggleButton>
   );
 
+  /**
+   * Renders font menu items
+   *
+   * @param font font
+   */
+  const renderFontMenuItem = (font: any) => {
```
Comment (Jdallos) [inline src/components/layout/v2/text-component-properties.tsx:264]: Could this be typed as string rather than any?

**Review by Jdallos — APPROVED** [review]

File: src/components/layout/v2/button-component-properties.tsx (hunk 1 above)
Comment (villejuutila) [reply src/components/layout/v2/button-component-properties.tsx]: Yes definitely. Resolved.

File: src/components/layout/v2/text-component-properties.tsx (hunk 2 above)
Comment (villejuutila) [reply src/components/layout/v2/text-component-properties.tsx]: Yes definitely. Resolved.

**Pull request merged**